amalgamate.sh

```diff
@@ -4,10 +4,9 @@ set -euox pipefail
 
 rust_dir=lighttpd_rust_amalgamated
 
-cmake -Wno-dev
-make clean
-bear -- make lighttpd
-make mod_{indexfile,dirlisting,staticfile}
+LIGHTTPD_STATIC=yes ./configure -C --enable-static=yes
+make -C src parsers versionstamp
+bear -- make -C src lighttpd
```

Comment:
```suggestion
make -C ./src parsers versionstamp
bear -- make -C ./src lighttpd
```

Reads a bit clearer I think since it differentiates the arguments.

Reply:
Do we need a `make clean` like before or no?